pkg/render/fluentd.go

```diff
@@ -395,8 +395,8 @@ func (c *fluentdComponent) packetCaptureApiRoleBinding() *rbacv1.RoleBinding {
 		Subjects: []rbacv1.Subject{
 			{
 				Kind:      "ServiceAccount",
-				Name:      ManagerServiceAccount,
```

Comment:
Just to double check, this was only here because the packet capture was originally attached to the manager right?

Reply:
Yes, that is correct.